blocks/utils/index.js

```diff
@@ -5,3 +5,56 @@
 export function getSiteUrl() {
 	return wpApiSettings.root.replace( '/wp-json/', '' );
 }
+
+/**
+ * Convert forms object in option
+ *
+ * @since 2.7.0
+ *
+ * @param {object} forms
+ *
+ * @return {[]}
+ */
+export function getFormOptions( forms ) {
+	let formOptions = [];
+
+	if ( forms ) {
+		formOptions = forms.map(
+			( { id, title: { rendered: title } } ) => {
+				return {
+					value: id,
+					label: title === '' ? `${ id } : ${ __( 'No form title' ) }` : title,
+				};
+			}
+		);
+	}
+
+	// Add Default option
+	formOptions.unshift( { value: '0', label: __( '-- Select Form --' ) } );
+
+	return formOptions;
+}
+
+/**
+ * Convert forms object in option
+ *
+ * @since 2.7.0
+ *
+ * @param {object} forms
+ * @param {number} SelectedFormId
+ *
+ * @return {boolean}
+ */
+export function isShowOldSettings( forms, SelectedFormId ) {
```

Comment:
Minor detail, but this function name is awkward and the description doesn't help. Is it returning whether a form is legacy or not?

Reply:
fixed: https://github.com/impress-org/givewp/compare/a7a47c648d73...2d98c0e71114